src/docs/sphinx/quickstart.rst

```diff
@@ -269,256 +269,71 @@ must be specified using either:
+Similarly, you need to set the ``operating_system`` to the proper value if you are not using ``sonoma`` (MacOS 14.X).
 
+If you want to install the devtools, you should also add the following under ``packages`` in the ``spack.yaml`` files.
 .. code-block:: yaml
-
+  # optional, for dev tools
   cppcheck:
-    version: [2.3]
+    version: [2.14.2]
     buildable: false
     externals:
-    - spec: cppcheck@2.3
-      prefix: /usr/local
+    - spec: cppcheck@2.14.1
+      prefix: /opt/homebrew
+  # optional, for dev tools
   doxygen:
-    version: [1.8.13]
-    buildable: false
-    externals:
-    - spec: doxygen@1.8.13
-      prefix: /usr/local
-  llvm:
-    version: [10.0.0]
-    buildable: false
-    externals:
-    - spec: llvm+clang@10.0.0
-      prefix: <path/to/llvm/10>
-  python:
+    version: [1.11.0]
     buildable: false
     externals:
-    - spec: python@3.9
-      prefix: <path/to/python/venv>
-
-Notes:
-
-* LLVM/Clang is needed for the style check tools. 
-  The *exact* version 10.0.0 is apparently highly recommended, since other versions may format the code slightly differently,
-  which will mean that pull requests formatted with them may trigger style errors in the CI checks.
-* The placeholders ``<path/to/llvm/10>`` and ``<path/to/python/venv>`` need to be filled in with actual paths.
-  See the following two notes. 
-* LLVM 10.0.0 has been superseded as the version for llvm-10; so this package is not easily installable with MacPorts.
-  You must build it yourself and then point to the build location.
-* For ``<path/to/python/venv>``, specify the the virtual environment directory created above.
+    - spec: doxygen@1.11.0
+      prefix: /opt/homebrew
 
 Building dependencies
 ^^^^^^^^^^^^^^^^^^^^^
 
-The invocation of ``uberenv.py`` is slightly modified from the standard instructions above in order to force the use of the
-MacPorts-installed MPI:
+The invocation of ``uberenv.py`` is slightly modified from the standard instructions above in order to force the use of the Homebrew-installed MPI and compilers:
 
 .. code-block:: bash
+   $ ./scripts/uberenv/uberenv.py --spack-env-file=scripts/spack/configs/macos_sonoma_aarch64/spack.yaml --prefix=../path/to/install --spec="%clang@18.1.8 ^openmpi@5.0.3_1"
 
-   $ ./scripts/uberenv/uberenv.py --prefix=../path/for/TPLs --spec="%clang@12.0.1 ^openmpi@4.1.4"
+Note: If you want to build with PETSc, you should instead use the command
 
-Notice the caret with the MPI spec. 
-Without this, current versions of Spack ignore the ``spack.yaml`` file and try to build a version of MPI from source.
-You can add additional specs as noted in the section `Building Serac's Dependencies via Spack/uberenv`.
+.. code-block:: bash
+   $ ./scripts/uberenv/uberenv.py --spack-env-file=scripts/spack/configs/macos_sonoma_aarch64/spack.yaml --prefix=../path/to/install --spec="++petsc %clang@18.1.8 ^openmpi@5.0.3_1 ^petsc+tetgen+scalapack+strumpack"
```

Comment:
```suggestion
   $ ./scripts/uberenv/uberenv.py --spack-env-file=scripts/spack/configs/macos_sonoma_aarch64/spack.yaml --prefix=../path/to/install --spec="+petsc %clang@18.1.8 ^openmpi@5.0.3_1 ^petsc+tetgen+scalapack+strumpack"
```

Reply:
The `++petsc` is to make sure MFEM also builds with PETSc, is that not needed?

Reply:
I can only confirm that ++ worked for me.  I have not tried without.

Reply:
`++` is the pass the variant on to all children. We should have handled that in our recipe. This is exposing a bug and I am surprised this didn't cause more problems.

There should be another line added here so this is not required:

https://github.com/LLNL/serac/blob/3f5eb149bedd2917eab124a8f73bc7560d5e9fcb/scripts/spack/packages/serac/package.py#L96-L100

Reply:
Ah, okay that makes sense. @chapman39 may have fixed that in #1138

Reply:
Yes he did. I would say to leave this like it is due to this being ready to go then have #1138 remove it because it wouldn't be needed anymore.